{"name": "Admin user Login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: invalid argument: invalid locator\n  (Session info: chrome=125.0.6422.60)\nStacktrace:\n\tGetHandleVerifier [0x00D40B13+45059]\n\t(No symbol) [0x00CD68A4]\n\t(No symbol) [0x00BD14FF]\n\t(No symbol) [0x00C118AF]\n\t(No symbol) [0x00C1210B]\n\t(No symbol) [0x00C4E022]\n\t(No symbol) [0x00C32DA4]\n\t(No symbol) [0x00C4BF64]\n\t(No symbol) [0x00C32AF6]\n\t(No symbol) [0x00C06948]\n\t(No symbol) [0x00C074CD]\n\tGetHandleVerifier [0x00FFF9A3+2924179]\n\tGetHandleVerifier [0x010536ED+3267549]\n\tGetHandleVerifier [0x00DC4474+584036]\n\tGetHandleVerifier [0x00DCB1DC+612044]\n\t(No symbol) [0x00CDFAB4]\n\t(No symbol) [0x00CDC358]\n\t(No symbol) [0x00CDC4F7]\n\t(No symbol) [0x00CCE46E]\n\tBaseThreadInitThunk [0x777CFA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x779175F4+228]\n\tRtlGetAppContainerNamedObjectPath [0x779175C4+180]\n\n", "trace": "  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test.py\", line 23, in login\n    fwu_login_functions.cancel_message(context.driver, fwu_login_elements.button_cancel_message)\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\Functions\\fwu_login_functions.py\", line 24, in cancel_message\n    driver.find_element('class', fwu_login_elements.button_cancel_message).click()\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 741, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "When to verify that user logged in with valid credential", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: invalid argument: invalid locator\n  (Session info: chrome=125.0.6422.60)\nStacktrace:\n\tGetHandleVerifier [0x00D40B13+45059]\n\t(No symbol) [0x00CD68A4]\n\t(No symbol) [0x00BD14FF]\n\t(No symbol) [0x00C118AF]\n\t(No symbol) [0x00C1210B]\n\t(No symbol) [0x00C4E022]\n\t(No symbol) [0x00C32DA4]\n\t(No symbol) [0x00C4BF64]\n\t(No symbol) [0x00C32AF6]\n\t(No symbol) [0x00C06948]\n\t(No symbol) [0x00C074CD]\n\tGetHandleVerifier [0x00FFF9A3+2924179]\n\tGetHandleVerifier [0x010536ED+3267549]\n\tGetHandleVerifier [0x00DC4474+584036]\n\tGetHandleVerifier [0x00DCB1DC+612044]\n\t(No symbol) [0x00CDFAB4]\n\t(No symbol) [0x00CDC358]\n\t(No symbol) [0x00CDC4F7]\n\t(No symbol) [0x00CCE46E]\n\tBaseThreadInitThunk [0x777CFA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x779175F4+228]\n\tRtlGetAppContainerNamedObjectPath [0x779175C4+180]\n\n", "trace": "  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test.py\", line 23, in login\n    fwu_login_functions.cancel_message(context.driver, fwu_login_elements.button_cancel_message)\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\Functions\\fwu_login_functions.py\", line 24, in cancel_message\n    driver.find_element('class', fwu_login_elements.button_cancel_message).click()\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 741, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\akbar.raza\\PycharmProjects\\FWU\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1715941445798, "stop": 1715941459821}, {"name": "Then user should be logged in successfully", "status": "skipped", "start": 1715941459822, "stop": 1715941459822}], "start": 1715941445797, "stop": 1715941459822, "uuid": "1a750121-21c7-4789-8f74-8a5064db79af", "historyId": "e3e7f1721e504f6b1c5a653a6a85665c", "fullName": "User login: Admin user Login", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "User login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}